crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -93,11 +89,8 @@ pub async fn daemon_server(
 
         let hash = format!("{}-turbo.log", base.repo_hash());
 
-        let logs = RelativeSystemPathBuf::new("logs").expect("forward relative");
-        let file = RelativeSystemPathBuf::new(hash).expect("forward relative");
-
-        let log_folder = folder.join_relative(logs);
-        let log_file = log_folder.join_relative(file);
+        let log_folder = folder.join_literal("logs");
+        let log_file = log_folder.join_literal(hash.as_str());
```

Comment:
Can we put the `format` call in here directly? I got worried that we weren't using something literal-ish i.e. user input.
```suggestion
        let log_file = log_folder.join_literal(&format!("{}-turbo.log", base.repo_hash()));
```

Reply:
Done.